components/invitation_modal/invitation_modal_guest_step/index.ts

```diff
@@ -32,4 +32,8 @@ function mapDispatchToProps(dispatch: Dispatch) {
     };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(InvitationModalGuestsStep);
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+export type PropsFromRedux = ConnectedProps<typeof connector>
+
+export default connector(InvitationModalGuestsStep);
```

Comment:
What do you think of this approach for redux auto inferring types

Reply:
Though one concern might be that there's no clear reference to what the types should be. Something like VSCode would likely be able to tell you, but not every editor would.

Reply:
@devinbinnie it is from official documentation, https://react-redux.js.org/using-react-redux/usage-with-typescript#inferring-the-connected-props-automatically
But anyways i dont want to bring in new coding styles all of a sudden, let me know if i should manually write the types.